modules/openapi-generator/src/main/resources/cpp-pistache-server/api-source.mustache

```diff
@@ -36,11 +36,13 @@ void {{classname}}::setupRoutes() {
 std::pair<Pistache::Http::Code, std::string> {{classname}}::handleParsingException(const std::exception& ex) const noexcept
 {
     try {
-        throw ex;
+        throw;
     } catch (nlohmann::detail::exception &e) {
         return std::make_pair(Pistache::Http::Code::Bad_Request, e.what());
     } catch ({{helpersNamespace}}::ValidationException &e) {
         return std::make_pair(Pistache::Http::Code::Bad_Request, e.what());
+    } catch (std::exception &e) {
+        return std::make_pair(Pistache::Http::Code::Internal_Server_Error, e.what())
```

Comment:
I'm new here but I think this needs a `;`?
`[ 96%] Building CXX object CMakeFiles/api-server.dir/model/User.cpp.o
/home/matt/src/openapi-generator/out/pis-petstore/api/StoreApi.cpp: In member function ‘virtual std::pair<Pistache::Http::Code, std::__cxx11::basic_string<char> > org::openapitools::server::api::StoreApi::handleParsingException(const std::exception&) const’:
/home/matt/src/openapi-generator/out/pis-petstore/api/StoreApi.cpp:54:85: error: expected ‘;’ before ‘}’ token
   54 |         return std::make_pair(Pistache::Http::Code::Internal_Server_Error, e.what())
`

Reply:
@stangerm2 thanks for catching the issue. I wonder if you can file a PR with the suggested fix?

Reply:
That is an oversight on my part. I did build the generated server, I must have removed the semicolon afterwards by accident?

I can also file a small PR to fix this later today.

Reply:
On another note, I don't think there is a pipeline that builds the generated example, or is there?

That could catch a mistake like this before it gets merged.